.github/workflows/doc.yml

```diff
@@ -29,11 +29,12 @@ jobs:
           echo "VERSION=$VERSION" >> $GITHUB_ENV
           echo "ID=$ID" >> $GITHUB_ENV
 
-      - name: Provision rdme
-        run: npm install -g rdme
-
       - name: Sync API spec
-        run: rdme openapi ./api/swagger/swagger.json --id=${{ env.ID }} --key=${{ secrets.README_TOKEN }}
+        uses: readmeio/rdme@v8
+        with:
+          rdme: openapi ./api/swagger/swagger.json --id=${{ env.ID }} --key=${{ secrets.README_TOKEN }}
```

Comment:
should we pass --version flag here?

Reply:
to have less risk on it I would add version flag as well

Reply:
Done.